content/2019/06/17/refactrtech.rst

```diff
@@ -0,0 +1,168 @@
+everyday lives, and everyone needs to be in on the conversation to really get a
+well-rounded perspective on what D&I is and how we can build better workplace
+cultures around this idea.
+
+Of course, women-focused events have their place, though some miss the mark on
+acknowledging intersectionality and understanding the unique challenges that
+say, queer and minority women face in the workplace. D&I is much more nuanced
+than just “we need more women.”
+
+We know that diversity makes good business sense and that companies that work
+towards gender, racial and ethnic diversity see financial gains above industry
+medians; but I love how the purview of this conference was less about how we can
+achieve D&I for financial gain and more about how being intentional about D&I
+can help technologists find purpose in the things we build.
+
+.. _Refactr.tech: http://refactr.tech/
+.. _Vue Vixens: https://vuevixens.org/
+.. _We RISE: https://www.womenwhocode.com/blog/women-who-code-announces-we-rise-tech-conference-in-atlanta
+
+Here's a recap.
+---------------
+
+There were the familiar formats for attendees to learn and make connections. Day
+one hosted full-day workshops on things like `GraphQL`_, `Flutter`_, `Gatsby`_
+and more. There was a `Vue Skulk`_, where I assume there were foxy Vue things
+going on. I’m sad to have missed the first day for mom things, but ran into an
+old co-worker who attended the Flutter workshop. It was his first experience
+with it, and in short-order he discovered that Flutter is indeed fun and easy to
+work with.
+
+.. _GraphQL: http://https://graphql.org/
+.. _Flutter: http://flutter.io
+.. _Gatsby: https://gatsbyjs.org/
+.. _Vue Skulk: https://vuevixens.org/
+
+Day two and three were dedicated to the standard keynotes and track sessions.
+`Tracy Lee`_ of `This Dot Labs`_ was on brand delivering the first keynote on
+how D&I makes good business sense, all while donning a prom dress. Day Two’s
+keynote took a more personal tone with `Dr. Kortney Ziegler`_ explaining how
+embracing his story has helped him find purpose in his endeavors at `ZaMLabs`_.
+
+.. _Tracy Lee: https://twitter.com/ladyleet
+.. _This Dot Labs: https://www.thisdot.co/labs
+.. _Dr. Kortney Ziegler: https://twitter.com/fakerapper?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor
+.. _ZaMLabs: https://sites.google.com/zamlabs.info/medialab/about
+
+Track sessions included Front-End Engineering, Software Engineering, Product &
+UX, Cool Sh!t, Career & Leadership, and Social Impact. I tried my best to get in
+on talks from each track. Speakers with various pathways into technology and
+levels of experience brought their own voices to present their passions, ideas,
+and tips and tricks.
+
+`Mike Harington`_ from the `Ionic`_ team went beyond the basics of what makes a
+website a progressive web app and demonstrated through code and personal
+experience which of the big three JavaScript frameworks offers the most support
+out of the box.
```

Comment:
oh, I want to hear more about the conclusion. (I saw him speak at VueConf, so I imagine that might be his preference?)

Reply:
Eh, I think he did lean towards Vue.  I was hoping the talks would be posted by now so that I could reference them.  I didn't expand on some of them since I didn't take the best notes.